sgkit/stats/aggregation.py

```diff
@@ -167,6 +200,53 @@ def count_variant_alleles(ds: Dataset, merge: bool = True) -> Dataset:
     return conditional_merge_datasets(ds, new_ds, merge)
 
 
+def count_cohort_alleles(ds: Dataset, merge: bool = True) -> Dataset:
+    """Compute per cohort allele counts from genotype calls.
+
+    Parameters
+    ----------
+    ds
+        Genotype call dataset such as from
+        `sgkit.create_genotype_call_dataset`.
+    merge
+        (optional)
```

Comment:
What should this `(optional)` indicate exactly?  I haven't been including it in many cases, but I don't know if we want it to mean:

1. It's a keyword-only argument
2. It's an argument with some default value
3. It can be none, like typing.Optional

Reply:
This shouldn't have been there, so I've removed it.